clients/src/main/resources/common/message/ApiVersionsResponse.json

```diff
@@ -45,7 +47,7 @@
       "about": "The duration in milliseconds for which the request was throttled due to a quota violation, or zero if the request did not violate any quota." },
     { "name":  "SupportedFeatures", "type": "[]SupportedFeatureKey", "ignorable": true,
       "versions":  "3+", "tag": 0, "taggedVersions": "3+",
-      "about": "Features supported by the broker.",
+      "about": "Features supported by the broker. Note: in v0-v3, features with MinSupportedVersion = 0 must be left out.",
```

Comment:
> Note: in v0-v3, features with MinSupportedVersion = 0 must be left out.

This needs to be adjusted accordingly since we now keep the feature but returns 1 as the minSupportedVersion.